index.d.ts

```diff
@@ -0,0 +1,35 @@
+import { HttpsAgent } from 'agentkeepalive';
+import { AgentOptions, Agent } from 'http';
+
+interface Logger {
+    info: () => void;
+    warn: () => void;
+    error: () => void;
```

Comment:
debug instead of error. we use it here https://github.com/vigneshshanmugam/agent-pool/blob/master/lib/agent-pool.js#L46

Reply:
I will add both

Reply:
cool